web/templates/personal_menu.hbs

```diff
@@ -0,0 +1,148 @@
+                {{/if}}
+
+            </div>
+            <div class="text-area">
+                <p class="full-name">{{user_full_name}}</p>
+                <p class="user-type">{{user_type}}</p>
+            </div>
+        </header>
+        <section class="popover-menu info_popover_actions" data-user-id="{{user_id}}">
+            <ul class="list">
+                {{#if status_content_available}}
+                <li class="text-item">
+                    <span>
+                        {{#if status_emoji_info}}
+                        {{#if status_emoji_info.emoji_alt_code}}
+                        <span class="emoji_alt_code">&nbsp;:{{status_emoji_info.emoji_name}}:</span>
+                        {{else if status_emoji_info.url}}
+                        <img src="{{status_emoji_info.url}}" class="emoji status_emoji" />
+                        {{else}}
+                        <span class="emoji status_emoji emoji-{{status_emoji_info.emoji_code}}"></span>
+                        {{/if}}
+                        {{/if}}
+                        <span class="status_text">
+                            {{status_text}}
+                        </span>
+                    </span>
+                    <a href="" class="clear_status" aria-label="{{t 'Clear status'}}" data-tippy-content="{{t 'Clear your status' }}">
+                        <i class="zulip-icon zulip-icon-x-circle"></i>
+                    </a>
+                </li>
+                <li class="link-item">
+                    <a href="" class="update_status_text">
+                        <i class="zulip-icon zulip-icon-smile"></i>
+                        {{#tr}}Edit status{{/tr}}
+                    </a>
+                </li>
+                {{else}}
+                <li class="link-item hidden-for-spectators">
+                    <a href="" class="update_status_text">
+                        <i class="zulip-icon zulip-icon-smile"></i>
+                        {{#tr}}Set status{{/tr}}
+                    </a>
+                </li>
+                {{/if}}
+
+                {{#if invisible_mode}}
+                <li class="link-item hidden-for-spectators">
+                    <a href="" class="invisible_mode_turn_off">
+                        <i class="zulip-icon zulip-icon-play-circle"></i>
+                        {{#tr}}Turn off invisible mode{{/tr}}
+                    </a>
+                </li>
+                {{else}}
+                <li class="link-item hidden-for-spectators">
+                    <a href="" class="invisible_mode_turn_on">
+                        <i class="zulip-icon zulip-icon-stop-circle"></i>
+                        {{#tr}}Go invisible{{/tr}}
+                    </a>
+                </li>
+                {{/if}}
+            </ul>
+
+            <ul class="list">
```

Comment:
Let's just use one `ul` tag here.

Reply:
So, I meant we should use one `ul` for the whole list and not multiple `ul` for each segment. We can use `.divider` elements as they are present in gear menu.